### PR TITLE
Test directory hygiene

### PR DESCRIPTION
**Pull request by jennybc**
@hadley I think this is non-controversial, yes?

It makes it much easier to get at the test specimens during development and the collection is growing.

### CODE REVIEW

**Comment by hadley** [comment]
Yeah, the only controversial thing is the use of rprojroot instead of `testthat::test_path()`, but I have a vague memory that there's some good reason that you can't do it

File: tests/testthat.R [hunk 1]
```diff
@@ -1,4 +1,10 @@
 library(testthat)
 library(readxl)
 
-test_check("readxl")
+if (requireNamespace("rprojroot")) {
```
Comment (hadley) [inline tests/testthat.R]: I think you can just skip this check - it's fine to assume that suggested packages are installed for testing

**Comment by jennybc** [comment]
> Yeah, the only controversial thing is the use of rprojroot instead of `testthat::test_path()`, but I have a vague memory that there's some good reason that you can't do it

At one time, `testthat::test_path()` did not work during `R CMD check`. But I just tried it and it works now, presumably due to https://github.com/hadley/testthat/commit/c83aba9169aa0c6ac3494dcb7a28fefa28ec051a. So once that's in CRAN version of testthat, I can switch over. I will leave myself a reminder.

File: tests/testthat.R (hunk 1 above)
Comment (jennybc) [inline tests/testthat.R]: Duly noted. For this and in future.

**Pull request merged**